app-misc/egads/egads-0.9.5-r2.ebuild

```diff
@@ -15,7 +15,17 @@ KEYWORDS="~amd64 ~ppc x86"
 
 EGADS_DATADIR=/var/run/egads
 
+PATCHES=(
+	"${FILESDIR}"/egads-0.9.5-make-build-work-with-clang16.patch
+)
+
+src_unpack() {
+	default
+}
+
 src_prepare() {
```

Comment:
You shouldn't need to call "aclocal", if you do, needs a die, but eautoreconf should include it anyway. But failing that, eaclocal.

Reply:
yes that is what I thought, but with eautoreconf it says
```
libtool: Version mismatch error.  This is libtool 2.4.7, but the
libtool: definition of this LT_INIT comes from an older release.
libtool: You should recreate aclocal.m4 with macros from libtool 2.4.7
libtool: and run autoconf again.
```
and build fails. 
With eaclocal same thing.

Reply:
Needed to `rm aclocal.m4 || die`. I checked it and it only had the libtool stuff in it, i.e. nothing homebrew (historically people used to have a huge aclocal.m4 file instead of m4/* for their own macros).